lib/editor/actions/editor.js

```diff
@@ -85,6 +85,11 @@ function getCloneProps (entityId: number, component: string, state: AppState) {
       // See https://github.com/catalogueglobal/datatools-ui/issues/153
       delete route.tripPatterns
       return route
+    case 'scheduleexception':
+      const scheduleExceptionClone = {
+        ...activeTable.find(e => e.id === entityId),
+        name: active.entity.name + ' (duplicate)'}
```

Comment:
You've got a `flow` error here. You can add an explicit `active.entity` && `active.entity.name` check above to fix this.

Reply:
`flow`